### PR TITLE
Add LOOT sorting

### PR DESCRIPTION
**Pull request by Silarn**
And CI / cleanup

### CODE REVIEW

File: .github/workflows/linting.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Lint Nehrim Plugin
```
Comment (Holt59) [inline .github/workflows/linting.yml]: :)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Build Nehrim Plugin
```
Comment (Holt59) [inline .github/workflows/build.yml]: ;)

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    runs-on: windows-2022
+    steps:
+      - name: Build Nehrim Plugin
+        uses: ModOrganizer2/build-with-mob-action@master
+        with:
+          mo2-third-parties: fmt gtest spdlog boost lz4
```
Comment (Holt59) [inline .github/workflows/build.yml]: Can be removed.

**Review by Holt59 — APPROVED** [review]
Looks good to me (not tested of course), except for the little CI comments.

**Comment by Silarn** [comment]
Right, I didn't change the CI names. I'll fix that shortly.

File: .github/workflows/build.yml (hunk 3 above)
Comment (Silarn) [reply .github/workflows/build.yml]: What do you mean, exactly? I don't think you want me to remove the entire build step.

Comment (Holt59) [reply .github/workflows/build.yml]: Sorry, I meant the `mo2-third-parties` - I have updated the build action to recursively fetch third-party dependencies from the `mo2-dependencies` list. 

Basically, you only need `mo2-third-parties` when the third-party dependency is specific to the plugin being build (or missing from the list in the build action).

Comment (Silarn) [reply .github/workflows/build.yml]: Okay, I see. I felt like those were still valid so it seemed odd to me, but I understand now. I'll tweak that soon.

**Pull request merged**